src/components/history-section/index.jsx

```diff
@@ -1,6 +1,62 @@
+import { addDefaultGroup, addHistory } from "../../firebase/CRUD";
 import IconButton from "../shared/IconButton";
 
 export default function HistorySection() {
+  let userToken = "";
+  const options = {
+    year: "numeric",
+    month: "numeric",
+    day: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    second: "numeric",
+  };
+  const currentDate = new Intl.DateTimeFormat("ko-KR", options).format(
+    new Date()
+  );
```

Comment:
P4: extension은 국제적으로 배포되니까, ISOString으로 저장하는 것을 어떨까요? 

```suggestion
  const currentDate = new Date().toISOString();
```

사용할 때는 `const currentDateObject = new Date(currentDate);` 와 같이 변환하는 것도 간편하고, 변환시 누락되는 정보도 없어서 의견남겨 봅니다. :)

Reply:
실제로 출력될 date를 사용자가 보기 편한 형태로 format한 상황이어서 toISOString()사용하면 문법적 오류가 발생합니다. 저도 국제표준시간을 기준으로 해야할지 많이 고민했었습니다. 우선 이대로 진행하고 말씀주신 부분에 대해서는 좀 더 고민해보고 차후에 반영하도록 하겠습니다.